### PR TITLE
[E2E] Add individual feature importance tests for responsibleaitoolbox-classification-model-assessment notebook

### PR DESCRIPTION
**Pull request by vinuthakaranth**
Adding e2e individual feature importance tests for responsibleaitoolbox-classification-model-assessment notebook.

### CODE REVIEW

File: apps/widget-e2e/src/describer/modelAssessment/Constants.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export enum locators {
+  IFIPredictionSpan = "span[class^='headerCount']", // IFI - Individual feature importance
+  IFIExpandCollapseButton = "[aria-label='expand collapse group']",
+  IFITableRowSelected = 'div[class^="ms-List-page"] div[class^="ms-DetailsRow"] div[class^="ms-Check is-checked"]',
+  IFIDropdownSelectedOption = "div[class^='featureImportanceChartAndLegend']",
+  IFINumberOfBars = "#FeatureImportanceBar svg .plot .points .point path",
+  IFIYAxisValue = '#FeatureImportanceBar div[class^="rotatedVerticalBox-"]',
+  IFIXAxisValue = '#FeatureImportanceBar g[class^="cartesianlayer"] g[class^="xtick"]',
+  ICEPlot = '#subPlotChoice label:contains("ICE")', // ICE - Individual conditional exception
```
Comment (imatiach-msft) [inline apps/widget-e2e/src/describer/modelAssessment/Constants.ts:9]: tiny nit: it's expectation, not exception, "ICE - Individual conditional exception" -> "Individual Conditional Expectation"
https://christophm.github.io/interpretable-ml-book/ice.html

File: apps/widget-e2e/src/util/Table.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        .parents(`${selectorPrefix} div[data-automationid='DetailsRow']`)
+        .click({ force: true });
+    });
+}
+
+export function getTableRowCount(srPrefix?: string): number {
+  const selectorPrefix = srPrefix ? `${srPrefix}` : "";
+  cy.get(
+    `${selectorPrefix} [data-automationid=DetailsList] [data-automationid=ListCell]`
+  ).should("exist"); // [data-automationid='DetailsList']
```
Comment (imatiach-msft) [inline apps/widget-e2e/src/util/Table.ts:27]: nit: remove comment
// [data-automationid='DetailsList']

**Review by imatiach-msft — APPROVED** [review]

File: apps/widget-e2e/src/util/Table.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function selectRow(
+  columnName: string,
+  columnValue: string,
+  srPrefix?: string
+): void {
+  const selectorPrefix = srPrefix ? `${srPrefix}` : "";
+  const columnsHeaderColumn = `${selectorPrefix} [data-automationid='ColumnsHeaderColumn']`;
```
Comment (xuke444) [inline apps/widget-e2e/src/util/Table.ts:10]: is this data-automationid from office fabric?
I am not seeing it is defined anywhere in our code.

Comment (vinuthakaranth) [reply apps/widget-e2e/src/util/Table.ts:10]: Yes, this is from office fabric

File: apps/widget-e2e/src/util/Table.ts (hunk 2 above)
Comment (vinuthakaranth) [reply apps/widget-e2e/src/util/Table.ts]: thank you, removed the commented code

File: apps/widget-e2e/src/describer/modelAssessment/Constants.ts (hunk 1 above)
Comment (vinuthakaranth) [reply apps/widget-e2e/src/describer/modelAssessment/Constants.ts]: thank you, updated.

**Pull request merged**